doc/support_matrix.yml

```
@@ -53,6 +53,18 @@ functionalities:
 
 # Encodes support status for each chip.
 chips:
+  nrf51822:
+    name: nRF51822
+    support:
+      gpio: supported
+      debug_output: supported
+      hwrng: not_currently_supported
+      i2c_controller: not_currently_supported
+      spi_main: not_currently_supported
+      logging: supported
+      storage: supported
```

```suggestion
      storage: not_currently_supported
```
To add support for storage, the `ariel-os-storage` build script needs to be modified and the `has_storage_support` laze module needs to be provided by the `nrf51` laze context.